modules/rollback-central-forced.adoc

```diff
@@ -28,7 +33,7 @@ $ oc -n stackrox rollout undo deploy/central <1>
 ----
 <1> If you use Kubernetes, enter `kubectl` instead of `oc`.
 ** To forcefully rollback to a specific version:
-. Edit Central’s `ConfigMap`:
+. Edit Central's `ConfigMap`:
```

Comment:
```suggestion
. Edit the `ConfigMap` that belongs to Central:
```

The IBM SG on possessives for inanimate objects (I guess it's not a living being!): 

![Screenshot From 2025-07-08 12-35-32](https://github.com/user-attachments/assets/ced13347-365d-4e3a-ac99-4d6bbafa1968)